go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/flatcar-linux/ignition
+
+go 1.16
+
+require (
+	github.com/ajeddeloh/go-json v0.0.0-20160803184958-73d058cf8437
+	github.com/aws/aws-sdk-go v1.8.39
+	github.com/coreos/go-semver v0.1.0
+	github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142
+	github.com/go-ini/ini v1.25.4 // indirect
+	github.com/godbus/dbus v0.0.0-20181025153459-66d97aec3384 // indirect
+	github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
```

Comment:
Can we do `sed -i '/indirect/d' go.mod`, it should get rid of indirect dependencies from here without side-effects.

Reply:
Mm, only a few of them are really useless (sed + go mod tidy):
```diff
diff --git a/go.mod b/go.mod
index 73da947e..9dedb7d3 100644
--- a/go.mod
+++ b/go.mod
@@ -7,20 +7,17 @@ require (
        github.com/aws/aws-sdk-go v1.8.39
        github.com/coreos/go-semver v0.1.0
        github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142
-       github.com/go-ini/ini v1.25.4 // indirect
-       github.com/godbus/dbus v0.0.0-20181025153459-66d97aec3384 // indirect
-       github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
+       github.com/go-ini/ini v1.63.2 // indirect
+       github.com/godbus/dbus v4.1.0+incompatible // indirect
+       github.com/jmespath/go-jmespath v0.4.0 // indirect
        github.com/pborman/uuid v0.0.0-20170612153648-e790cca94e6c
        github.com/pin/tftp v2.1.0+incompatible
-       github.com/sigma/bdoor v0.0.0-20160202064022-babf2a4017b0 // indirect
+       github.com/sigma/bdoor v0.0.0-20160624225336-b9c82b7b3c0b // indirect
        github.com/sigma/vmw-guestinfo v0.0.0-20160204083807-95dd4126d6e8
-       github.com/smartystreets/goconvey v1.7.2 // indirect
        github.com/stretchr/testify v1.7.0
        github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb
-       github.com/vmware/vmw-guestinfo v0.0.0-20170707015358-25eff159a728 // indirect
+       github.com/vmware/vmw-guestinfo v0.0.0-20211006225857-cc1fd90d572c // indirect
        github.com/vmware/vmw-ovflib v0.0.0-20170608004843-1f217b9dc714
        go4.org v0.0.0-20160314031811-03efcb870d84
        golang.org/x/net v0.0.0-20190320064053-1272bf9dcd53
-       golang.org/x/text v0.3.1-0.20190321115727-fe223c5a2583 // indirect
-       gopkg.in/yaml.v2 v2.4.0 // indirect
```

Reply:
Fair enough, I'm not sure how this works exactly either. Resolving anyway.